_pages/en/strategy-learning-automating-improving.md

```diff
@@ -54,68 +55,20 @@ IITB teams should:
 ##### Learn
 
 - Stay up to date on trends and technologies inside and outside the GC
-- Teach yourself basic programming. Examples of instructional material can be found [here](#Learn).
-- Experiment with cloud technologies. Click [here](#Learn) to learn how to create a cloud sandbox environment.
-  - Ex: [Cousera](https://www.coursera.org/), [Udemy](https://www.udemy.com/), or [MOOCs](https://www.mooc-list.com/)
-- Read books, articles or documentation. Examples of content can be found [here](https://sara-sabr.github.io/ITStrategy/references-en.html).
+- Teach yourself basic programming
+- Experiment with cloud technologies
+- Take online courses - [edX](https://www.edx.org/), [Cousera](https://www.coursera.org/), [Udemy](https://www.udemy.com/), or [MOOCs](https://www.mooc-list.com/)
+- Read books, articles or documentation - See our [References]({{ site.baseurl }}/references-en.html#Q1)
 - Attend events or conferences
 - Request micro missions with groups throughout ESDC or the Government of Canada at large
-- Attend [IITB showcase](#Learn), [Dev CoP](#Learn), dojos or other learning opportunities within IITB and ESDC
-- Participate in "Innovation" at ESDC and [Innovation Lab](#Learn) design sessions
+- Attend IITB showcase, Dev CoP, dojos or other learning opportunities within IITB and ESDC
```

Comment:
Make it easy for readers by adding hyperlinks:
IITB Showcase: http://dialogue/grp/BU6386699/SitePages/IITB_Showcase.aspx
Dev CoP: https://github.com/esdc-edsc/Welcome/blob/master/Recommendations/DevOps_SDLC.md
Dojos: (will need to contact Mike Murphy's team)